File: 3718-smallest-missing-multiple-of-k/3718-smallest-missing-multiple-of-k.cpp
class Solution {
public:
    int missingMultiple(vector<int>& nums, int k) 
    {
        set<int> num(nums.begin(),nums.end());
        int x=k;
        while(num.find(x)!=num.end())
        {
            x+=k;
        }
        return x;
    }
};

File: 3718-smallest-missing-multiple-of-k/3718-smallest-missing-multiple-of-k.py
class Solution(object):
    def missingMultiple(self, nums, k):
        """
        :type nums: List[int]
        :type k: int
        :rtype: int
        """
        num=set(nums)
        x=k
        while x in num:
            x+=k
            
        return x









